packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewer.js

```diff
@@ -277,7 +277,7 @@ class LightCurveViewer extends Component {
   onAnnotationBrushEnd (annotationBrush, index, domElements) {
     const props = this.props
     const brushSelection = d3.event.selection // Returns [xMin, xMax] or null, where x is relative to the SVG (not the data)
-    const defaultBrush = this.getDefaultBrush()
+    this.getDefaultBrush()
```

Comment:
I think this can just be removed, as it's not doing anything with the return value

Reply:
Sounds good if the LCV works without it.